components/Avatar/Avatar.js

```diff
@@ -10,6 +10,7 @@ export function Avatar({
   animate,
   colorRing,
   isUnread = true,
+  link = `/stories/${user.user_id}`,
```

Comment:
What if the user has no stories?

Reply:
This code is from an old commit.
This issue was fixed in a newer commit.
https://github.com/alshakerM/my-instagram/pull/83/commits/767952d3d093835b725e014942742c609831abc0